### ISSUE TITLE
Snapdragon 845

### ISSUE THREAD

**Issue by AdyAndrews** [Vita3K/Vita3K-Android#221]
Ignore these clowns please fix issue on snapdragon 845 first

**Comment by DigiXplora** [comment]
You think your problem is special? This is the same attitude you guys started and drove away AetherSx2 developers, clown 🙄

**Comment by AdyAndrews** [comment]
> 

Yes my problem is "very" special cuz no game is running on my device  and what attitude u talking about I didn't said anything to aethersx2 developer infact I'm new here , clown 😏

**Comment by AdyAndrews** [comment]
And people like u should mind your own damn business I'm talking to makdu here

**Comment by DigiXplora** [comment]
Go get a better phone, clown.

**Comment by AdyAndrews** [comment]
Ask your mom to buy me a phone clown or should I say bloody Indian
